### PR TITLE
Throw cron command to /dev/null

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -42,7 +42,7 @@
   $policies_yaml = inline_template('<%= Hash[policies_array.map { |p|
       ["foreman_scap_client_#{p["id"]}",
         {
-          "command" => "/usr/bin/foreman_scap_client #{p[\'id\']}",
+          "command" => "/usr/bin/foreman_scap_client #{p[\'id\']} > /dev/null",
```
Comment (ares) [inline manifests/init.pp:45]: you don't want to send mail notifications? if so, maybe you should also redirect 2> /dev/null

**Comment by cristifalcas** [comment]
I think errors should be sent by email, but not normal output. Can this be merged?

**Comment by ares** [comment]
@cristifalcas so you'd prefer built-in mail notification system (which we currently don't have in foreman-scap-client) over cron capabilities (MAILTO variable)? I think this should be configurable defaulting to use `&> /dev/null`

**Comment by cristifalcas** [comment]
No, I was saying that redirecting only stdout to /dev/null is correct, because everything appearing on stderr will be sent by mail by cron itself.

**Comment by ares** [comment]
The thing is, that the foreman_scap_client puts both debugging and error logs to STDOUT, should that be fixed first then?

**Comment by cristifalcas** [comment]
Setting &> /dev/null will hide errors even now: program missing, for example (and it will also hide them when they will be available on stderr). The current configuration (no redirect) is sending emails (200 in our case) on each cron run, so we don't want that either. For cron jobs only stdout should be sent to /dev/null or a log file.

**Comment by ares** [comment]
ok, sorry for the delay, @shlomizadok consider this ACKed, just please change the commit message so it matches redmine issue http://projects.theforeman.org/issues/9994

**Pull request merged**